### ISSUE TITLE
Contrast ratio for Oppia icon is not as per the standards

### ISSUE THREAD

**Issue by KolliAnitha** [oppia/oppia-android#4316]
**Describe the bug**
The Contrast ratio for Oppia icon is not as per the standards

**To Reproduce**
1. Install Oppia Android application
2. Go to Home and enable accessibility scanner
3. Verify the results

**Expected behavior**
The contrast ratio should be 4.5:1

Actual behavior:
The contrast ratio for icon is 4.15:1
![Screenshot 2022-04-25 2 42 13 PM](https://user-images.githubusercontent.com/101634267/165058490-46f0cfe1-0035-479f-9521-771ba8299f2d.png)

**Environment**
Device name: One plus Nord2 5G
Android version : Android 11
App version : 0.7-alpha-d1b2827517

**Comment by BenHenning** [comment]
The icon isn't finalized yet, but this is a good item to catalog as part of beta/GA.

**Comment by BenHenning** [comment]
For searching context, this issue was found in 0.7-alpha (MR4).

**Comment by BenHenning** [comment]
@KolliAnitha does this still happen with the latest beta version of the icon?

**Comment by KolliAnitha** [comment]
@BenHenning, tested the contrast ratio. It still fails
![Screenshot_2022-09-15-07-16-11-89_80a9469f53bfe75dbc6e8989be28e2f0.jpg](https://user-images.githubusercontent.com/101634267/190293974-dab7ec0d-0056-44a5-add6-b53aa28d8fbb.jpg)

**Comment by BenHenning** [comment]
Thanks for checking @KolliAnitha.

This will need UX guidance for how to proceed. We may need to consider alternate colors at this point to ensure correct contrast.

**Comment by seanlip** [comment]
Hi @KolliAnitha, this is actually an issue with the contrast between the text and the user's background on the screen (look at the two colours in the contrast checker). Note that the check being applied here is for "text contrast".

Since we do not have control over the user's background, it's not feasible to fix this, so I'm closing this issue.
